src/control/server/mgmt_pool_test.go

```diff
@@ -213,14 +212,13 @@ func TestServer_MgmtSvc_calculateCreateStorage(t *testing.T) {
 			},
 			expOut: &mgmtpb.PoolCreateReq{
 				Tierbytes: []uint64{defaultScmBytes / 2, defaultNvmeBytes / 2},
-				Tierratio: []float64{0, 0},
 				Ranks:     []uint32{0, 1},
 			},
 		},
 		"auto sizing (not enough SCM)": {
 			in: &mgmtpb.PoolCreateReq{
 				Totalbytes: scmTooSmallTotal,
-				Tierratio:  []float64{scmTooSmallRatio},
+				Tierratio:  []float64{scmTooSmallRatio, 0},
```

Comment:
what happens if you don't provide the nvme tier?

Reply:
that's not possible because of the logic in `dmg/pool.go:storageManual()` and validation in `lib/control/pool.go:poolCreateReqChkSizes()`

Reply:
i.e. the validation in both those functions constricts the possible combinations so missing nvme tier would be handled early. Possible I'm not understanding the context of your question correctly though.